contracts/sentinel/guards/FuseWithdrawalGuard.sol

```diff
@@ -22,7 +22,7 @@ contract FuseWithdrawalGuard is IGuard, CoreRef {
     EnumerableSet.AddressSet private fuseDeposits;
 
     /// @notice the PCV mover contract exposed to guardian role
-    PCVGuardian public constant pcvGuardian = PCVGuardian(0x02435948F84d7465FB71dE45ABa6098Fc6eC2993);
+    PCVGuardian public constant pcvGuardian = PCVGuardian(0x0000000000000000000000000000000000000000); // TODO update
```

Comment:
Commenting to remind of todo.

Reply:
thanks can only be fixed after deployment

Reply:
This guardian address should be injected in at constructor time no? Otherwise I imagine it's impossible to test this